### DISCUSSION TITLE
Vercel free tier bandwidth youtube

### DISCUSSION THREAD

**Discussion by barcell1** [vercel/community#529] [Help] [answered]
if i put youtube videos on my vercel site , i spend my free tier bandwith with the streaming? if yes gonna be huge ?

**Comment by leerob** [comment, marked as answer]
If you're embedding a YouTube video as an iframe, there won't be additional bandwidth from the video, since it's not hosted on Vercel. You would only have bandwidth from loading that page, not the video files.

https://vercel.com/support/articles/best-practices-for-hosting-videos-on-vercel-nextjs-mp4-gif

**Reply by barcell1** [reply]
thanks for the answer, but just to see if i understand, the best practice to import a video from youtube on vercel is using <iframe/>

**Reply by leerob** [reply]
Yeah!
